docs/content/python/getting-started/quickstart/index.md

```diff
@@ -42,22 +40,24 @@ role-based access control.
 
 {{% exampleGet "import_code" %}}
 
+To refer to your {{% exampleGet "classes" %}} in Polar, you must _register_ them with Oso.
+
+{{% exampleGet "register_classes" %}}
+
 ## Accept or deny requests
 
-When a request arrives, your application will need to ask Oso if it should accept the request. Oso needs three pieces of information to make that decision:
+Oso needs three pieces of information to make an authorization decision:
 - Who is making the request? (the "actor")
 - What are they trying to do? (the "action")
 - What are they doing it to? (the "resource")
 
 You'll pass these pieces of information are to Oso's `{{% exampleGet "isAllowed" %}}` method: `{{% exampleGet "isAllowed" %}}(actor, action, resource)`.
```

Comment:
into Oso's

Reply:
I don't quite understand this one. Where should that be inserted?

Reply:
L54

You'll pass these pieces of information **into** Oso's